main.go

```diff
@@ -54,13 +55,17 @@ func HandleConfigFile(filePath string) (config.ConfigReader, error) {
 
 	c := viper.New()
 
+	// map settings such as foo.bar to FOO_BAR environment keys
+	c.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+
 	//Allow AWS URI endpoint to be set by environment variable
 	c.BindEnv(SettingAwsURI, "AWS_URI")
 
 	//Allow aws keyid, aws token, aws secret to be read by viper
 	c.BindEnv(SettingAwsAuthKeyId, "AWS_ACCESS_KEY_ID")
 	c.BindEnv(SettingAwsAuthSecret, "AWS_SECRET_ACCESS_KEY")
 	c.BindEnv(SettingAwsAuthToken, "AWS_SESSION_TOKEN")
+	c.BindEnv(SettingAwsS3Region, "AWS_S3_REGION")
```

Comment:
with MEN-747 all of those supposed to get thrown away, not sure if there is a point introducing this now